src/expressServer.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+//const path = require('path');
```

Comment:
Can be removed?

Reply:
I'm leaving this in because the code below was removed by mistake